doc/api/punycode.md

```diff
@@ -1,6 +1,11 @@
 # punycode
 
-    Stability: 2 - Stable
+    Stability: 0 - Deprecated
+
+**The version of the punycode module bundled in Node.js is being deprecated**.
+In a future major version of Node.js this module will be removed. Users
```

Comment:
Should we use more specific version number instead of the word "future"?

Reply:
https://nodejs.org/api/punycode.html